sieve/script.c

```diff
@@ -430,7 +430,7 @@ static int build_notify_message(sieve_interp_t *i,
 static int send_notify_callback(sieve_interp_t *interp,
                                 void *message_context,
                                 void *script_context, notify_list_t *notify,
-                                char *actions_string,
+                                char *actions_string __attribute__((unused)),
```

Comment:
Do we want to just remove the `actions_string` parameter all together rather than just commenting out the place where it is used?

Reply:
Yes, probably. :)

Reply:
Done.